dht-node/src/dht_node/index.ts

```diff
@@ -203,7 +218,56 @@ async function writeFederatedHomeCommunityEnries(pubKey: any): Promise<Community
       logger.info(`federation home-community inserted successfully: ${JSON.stringify(homeCom)}`)
     })
   } catch (err) {
-    throw new Error(`Federation: Error writing HomeCommunity-Entries: ${err}`)
+    throw new Error(`Federation: Error writing federated HomeCommunity-Entries: ${err}`)
   }
   return homeApiVersions
 }
+
+async function writeHomeCommunityEntry(pubKey: any): Promise<void> {
```

Comment:
would be nice to have the type of pubKey here to not use any

Reply:
yes it would be, but the return of the DHT.keyPair() returns no types but simply any...